src/include/parser/parsed_expression_visitor.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <unordered_map>
+
+#include "parser/expression/parsed_expression.h"
+
+namespace kuzu {
+namespace parser {
+
+class ParsedExpressionChildrenCollector {
```

Comment:
Merge `ParsedExpressionChildrenCollector` and `ParsedExpressionChildrenSetter` as a `ParsedExpressionChildrenVisitor`